src/EnergyPlus/Furnaces.cc

```diff
@@ -165,8 +165,6 @@ namespace Furnaces {
     // MODULE PARAMETER DEFINITIONS
     static constexpr std::string_view BlankString;
 
-    constexpr std::string_view fluidNameSteam("STEAM");
```

Comment:
There is a direct shortcut now for getting the steam fluid properties object that does not require a lookup by name.